src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable no-console */
+import { useContext } from 'react';
+// eslint-disable-next-line import/no-cycle
+import { TodoItem } from '../TodoItem/TodoItem';
+import { Status } from '../../types/Status';
+import { filterByStatus } from '../../services/filterByStatus';
+
+// eslint-disable-next-line import/no-cycle
+import {
+  TodosContext,
+} from '../../TodosContext/TodosContext';
+
+type Props = {
```

Comment:
use interface prefer. 
Types have unique features but the interface is best for scaling.